frontend/taipy-gui/src/components/Taipy/Progress.spec.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021-2024 Avaiga Private Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+import Progress from "./Progress";
+
+describe("Progress component", () => {
+    it("renders circular progress without value (indeterminate)", () => {
+        render(<Progress />);
```

Comment:
you can get the getByRole and getBy* functionc from the render like it's done in the other tests
const { getByRole } = render(...)